sharding-jdbc-example/sharding-example/sharding-raw-jdbc-example/src/main/java/org/apache/shardingsphere/example/sharding/raw/jdbc/JavaConfigurationExample.java

```diff
@@ -44,9 +46,17 @@ public static void main(final String[] args) throws SQLException {
         commonService.initEnvironment();
         commonService.processSuccess();
         commonService.cleanEnvironment();
+        CommonService countryService = getCountryService(dataSource);
+        countryService.initEnvironment();
```

Comment:
we should only use countryService here, commonService can be removed.

Reply:
ok